src/platform/silabs/rs911x/BLEManagerImpl.cpp

```diff
@@ -60,6 +60,8 @@ extern "C" {
 #include <setup_payload/AdditionalDataPayloadGenerator.h>
 #endif
 
+#define RSI_BLE_ADDR_MAX_LENGTH 6
```

Comment:
I suggest using constexpr instead of define (it is more in line with our c++ standard). I also believe that define exists somewhere else